compiler/rustc_resolve/src/macros.rs

```diff
@@ -1139,14 +1139,16 @@ impl<'ra, 'tcx> Resolver<'ra, 'tcx> {
         }
     }
 
-    pub(crate) fn check_reserved_macro_name(&self, ident: Ident, res: Res) {
+    pub(crate) fn check_reserved_macro_name(&self, ident: ModernIdent, res: Res) {
```

Comment:
```suggestion
    pub(crate) fn check_reserved_macro_name(&self, ident: Ident, res: Res) {
```
This function would take a `Symbol` if it not needed the span for the error.
In general, for displaying spans non-normalized idents are better.